assets/css/order-styles.css

```diff
@@ -175,3 +175,6 @@
 	0% { transform: rotate(0deg); }
 	100% { transform: rotate(-360deg); }
 }
+.pdf-hide {
```

Comment:
I think you could avoid using this class by hiding the elements with JS.

Reply:
I have used this class to hide the details div and 'Hide details' link on page load. If there is delay in js loading due slow network or some other reason, then it will first display all the details and then it will hide elements after js is loaded. There will be a display glitch.

Reply:
I understand, good point. Just include the instruction directly in the element then, without using the class. I don't think it worth having a class just for that.